index.html

```diff
@@ -1,62 +1,246 @@
+            <div class="description"> 
+          <h3>Multi-Post Stories <br> Gain+Glory</h3>
+          <ul class="project-languages">
+            <li class="lang-rr">Ruby on rails</li>
+            <li class="lang-cs">css</li>
+            <li class="lang-js">JavScript</li>
+            <li class="lang-ht">html</li>
+          </ul>
+          <button type="button" class="btn-card"> See Project</button>
+          </div>
+        </article>
+
+        <article class="card-one">
+          
+            <img src="images/ImgPlaholder.png" alt="image holder" class="imgholder">
+            <div class="description">
+          <h3>Multi-Post Stories <br> Gain+Glory
+
+          </h3>
+          <ul class="project-languages">
+            <li class="lang-rr">Ruby on rails</li>
+            <li class="lang-cs">css</li>
+            <li class="lang-js">JavScript</li>
+            <li class="lang-ht">html</li>
+          </ul>
+          <button type="button" class="btn-card"> See Project</button>
+          </div>
+        </article>
+
+        <article class="card-one">
+          
+            <img src="images/ImgPlaholder.png" alt="image holder" class="imgholder">
+            <div class="description">
+          <h3>Multi-Post Stories <br> Gain+Glory
+
+          </h3>
+          <ul class="project-languages">
+            <li class="lang-rr">Ruby on rails</li>
+            <li class="lang-cs">css</li>
+            <li class="lang-js">JavScript</li>
+            <li class="lang-ht">html</li>
+          </ul>
+          <button type="button" class="btn-card"> See Project</button>
+          </div>
+        </article>
+
+        <article class="card-one">
+
+          <img src="images/ImgPlaholder.png" alt="image holder" class="imgholder">
+          <div class="description">
+          <h3>Multi-Post Stories <br> Gain+Glory
+          </h3>
+          <ul class="project-languages">
+            <li class="lang-rr">Ruby on rails</li>
+            <li class="lang-cs">css</li>
+            <li class="lang-js">JavScript</li>
+            <li class="lang-ht">html</li>
+          </ul>
+          <button type="button" class="btn-card"> See Project</button>
+        </div>
+        </article>
+
+        <article class="card-one">
+          
+            <img src="images/ImgPlaholder.png" alt="image holder" class="imgholder">
+            <div class="description">
+          <h3>
+            Multi-Post Stories <br> Gain+Glory
+
+          </h3>
+          <ul class="project-languages">
+           
+            
+            <li class="lang-rr">Ruby on rails</li>
+            <li class="lang-cs">css</li>
+            <li class="lang-js">JavScript</li>
+            <li class="lang-ht">html</li>
+            
+          </ul>
+          <button type="button" class="btn-card"> See Project</button>
+        </div>
+        </article>
+      </div>
+
+    </section>
+
+    
+
+    <section class="about_section">
```

Comment:
- [ ] Your background images are not displaying. Please take measures to fix this issue, especially for the about section, as for this milestone you have to implement the work and about section. But It is highly recommended to implement it for the headline too.